frontend/src/components/MapView/DateSelector/TimelineItems/TimelineItem/index.tsx

```diff
@@ -32,7 +32,7 @@ const TimelineItem = memo(
     ): boolean => false;
 
     const isQueryDate = (date: DateItem): boolean =>
-      date.queryDate === date.displayDate;
+      datesAreEqualWithoutTime(date.queryDate, date.displayDate);
```

Comment:
Note - The 12h shift is also used to avoid timezone errors and avoid differences between what would get displayed for two users in opposite timezones. We would need to check that this does not introduce unexpected behaviors.

Reply:
Ran this under various timezone on my computer (london, tokyo, manila) and it all looks good 👍 Definitely a good thing check, thanks for the reminder